docs/source/models/cause.rst

```diff
@@ -558,6 +558,85 @@ below.
 Remission Rates
 ^^^^^^^^^^^^^^^
 
+**Definition**
+
+Remission is a measure of cases that recover from a with-condition state, given 
+a specified population and time period. Just as with incidence, the numerator is 
+given by the count of recovered (or *remitted*) cases, and the denominator is 
+the cumulative person-time [link to person-time definition in incidence 
+section] during which cases are able to go into remission.
+
+  For example, consider diarrhea cases in the Philippines in 2017. Say that that 
+  year, every such case remitted after an average of 5 days:
+
+  .. math::
+
+    \frac{\text{1 case}}{\text{5 person-days}}=\frac{365/5\text{ cases}}{\text{1 person-year}}=73\text{ cases/person-year}
+
+  This calculation is straightfoward, as diarrheal diseases have a highly
+  consistent disease duration.
+
+  In contrast, consider diabetes. Say that there were 142,794 prevalent cases of
+  diabetes (both type I and type II) in Moldova amongst males in 2017, and of 
+  those 142,794 cases, 509 remitted in 2017. This gives us the following rate:
+
+  .. math::
+
+    \frac{\text{509 cases}}{\text{142,794 person-years}} =
+    \frac{\text{0.0036 cases}}{\text{1 person-year}}
+
+  It is important to note that this is a remission rate for diabetes at all
+  ages. In the context of the diarrheal diseases example, this makes it seem as 
```

Comment:
Why is it important to note that this is a remission rate for diabetes at all ages? (And, for example, why was it not important to note this for diarrhea? Presumably because remission rates vary significantly across ages for diabetes, but not for diarrhea?) It's just a bit unclear how this sentence fits in with the rest of the discussion, since then you specifically discuss type I vs. type II, but don't mention ages again.

In general, though I think this new version is much clearer.